src/Mentat.UI/wwwroot/js/Student.js

```diff
@@ -199,4 +208,101 @@ function rebuildCarousel(selectedTags) {
+function getUserDifficultyLevel(currentCardId)
+{
+    var userRating = JSON.parse(localStorage.getItem(JSON.stringify(currentCardId)));
+    setUserRatingButtonColor(userRating);
+    
+}
+
+// Calculate the users score
+function getUserScore() {
+    var score = 0.0;
+    for (var i = 0; i < localStorage.length; i++) {
+        var r = JSON.parse(localStorage.getItem(localStorage.key(i)))
+        if (r == 'yes') {
+            score = score + 1.0;
+        }
+        else if (r == 'partial') {
+            score = score + 0.5
+        }
+    }
+    let userScore = document.getElementById('score');
+    userScore.innerText = score;
+}
+
+// Set the button color based on the user response
+function setUserRatingButtonColor(ur) {
+    switch (ur) {
+        case 'yes':
+            document.getElementById('yesButton').style.backgroundColor = 'green';
+            document.getElementById('yesButton').style.color = 'white';
+            document.getElementById('partialButton').style.backgroundColor = '#CEB888';
+            document.getElementById('partialButton').style.color = 'black';
+            document.getElementById('noButton').style.backgroundColor = '#CEB888';
+            document.getElementById('noButton').style.color = 'black';
+            break;
+        case 'partial':
+            document.getElementById('yesButton').style.backgroundColor = '#CEB888';
+            document.getElementById('yesButton').style.color = 'black';
+            document.getElementById('partialButton').style.backgroundColor = 'yellow';
+            document.getElementById('partialButton').style.color = 'black';
+            document.getElementById('noButton').style.backgroundColor = '#CEB888';
+            document.getElementById('noButton').style.color = 'black';
+            break;
+        case 'no':
+            document.getElementById('yesButton').style.backgroundColor = '#CEB888';
+            document.getElementById('yesButton').style.color = 'black';
+            document.getElementById('partialButton').style.backgroundColor = '#CEB888';
+            document.getElementById('partialButton').style.color = 'black';
+            document.getElementById('noButton').style.backgroundColor = 'red';
+            document.getElementById('noButton').style.color = 'black';
+            break;
+        default:
+            document.getElementById('yesButton').style.backgroundColor = '#CEB888';
+            document.getElementById('yesButton').style.color = 'black';
+            document.getElementById('partialButton').style.backgroundColor = '#CEB888';
+            document.getElementById('partialButton').style.color = 'black';
+            document.getElementById('noButton').style.backgroundColor = '#CEB888';
+            document.getElementById('noButton').style.color = 'black';
+            break;
+    }
+}
+
+// Allow the arrow keys to move the cards left/right and the Enter key to show/hide the answer
```

Comment:
I have added a title to each of the arrows and the show/hide button. I did originally use the space bar instead of the enter key, but pressing the space bar on my keyboard moves the window up instead. I will continue looking into this.